src/cascade/input_data/configuration/form.py

```diff
@@ -199,6 +199,7 @@ class CountryCovariate(Form):
 class Model(Form):
     modelable_entity_id = IntField()
     model_version_id = IntField(nullable=True)
+    random_seed = IntField()
```

Comment:
Should this go into the options table during serialization? It probably should.

Reply:
Yes. Yes, it should. As should the zero_sum_random. We've missed a few tricks.